hrm/department_app/models/employees.py

```diff
@@ -0,0 +1,18 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Employees(models.Model):
```

Comment:
Please use the singular form when naming models.

Reply:
Fixed.